### PR TITLE
Test

### PR DESCRIPTION
**Pull request by nkutsubos**

### CODE REVIEW

File: i18n/plugins/org.talend.designer.core.nl/src/main/java/messages_fr.properties [hunk 1]
```diff
@@ -84,7 +84,7 @@ SqlMemoController.QueryError.mainMsg=Erreur de requ\u00EAte.
 ContextRepositoryCommand.modifyContext=Modifier un contexte
 EParameterName.DisableWarnings=D\u00E9sactiver les alertes
 ChangeConnTextCommand.Label=Modifier le libell\u00E9 de la connexion
-TalendEditorPaletteFactory.Note=Note
+TalendEditorPaletteFactory.Note=Remarque\u00A0:
```
Comment (nrousseau) [inline i18n/plugins/org.talend.designer.core.nl/src/main/java/messages_fr.properties]: Don't think it shoudn't be modified, it's a "Note" component, available for the palette for the user.
Even more with :

File: i18n/plugins/org.talend.designer.runprocess.nl/src/main/java/messages_fr.properties [hunk 2]
```diff
@@ -1,7 +1,7 @@
 ProcessComposite2.statsComposite=Stats && Traces
 ProcessComposite.clearBefore=Effacer avant l\'ex\u00E9cution
 ProcessComposite.kill=Arr\u00EAter
-ProcessComposite.confirmText=Voulez-vous vraiment lancer le Job avec le contexte {0} ?
+ProcessComposite.confirmText=Voulez-vous lancer le Job avec le contexte {0}\u00A0?
```
Comment (pteyssier) [inline i18n/plugins/org.talend.designer.runprocess.nl/src/main/java/messages_fr.properties]: Why some spaces are replaced by unicode point?

File: i18n/plugins/org.talend.designer.core.nl/src/main/java/messages_fr.properties (hunk 1 above)
Comment (sbouyer) [reply i18n/plugins/org.talend.designer.core.nl/src/main/java/messages_fr.properties]: Changed! thx

File: i18n/plugins/org.talend.designer.runprocess.nl/src/main/java/messages_fr.properties (hunk 2 above)
Comment (sbouyer) [reply i18n/plugins/org.talend.designer.runprocess.nl/src/main/java/messages_fr.properties]: It is a non-breaking space.

**Pull request closed without merging**